### ISSUE TITLE
Pan causes scroll jump

### ISSUE THREAD

**Issue by leesei** [openseadragon/openseadragon#954]
To reproduce this issue:
- have the OSD canvas partially shown on the page with scroll
  ![screenshot-2016-06-05-224524](https://cloud.githubusercontent.com/assets/544400/15806170/163ef618-2b70-11e6-9b5e-1d887c666f77.png)
- interact with the canvas with mouse (I fling to pan it to upper-right)
- the scroll jumps so the bottom of the canvas meets the bottom of client window (or the top meets the top of client window if the canvan cannot be shown completely)
  ![screenshot-2016-06-05-224549](https://cloud.githubusercontent.com/assets/544400/15806186/9157807c-2b70-11e6-8a23-6f43afa7d930.png)

I tracked down to `canvas.focus()` call in `onCanvasClick()`
https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L2449-L2451
I verified commenting the line would solve the scroll jump issue.

Should we check if key navigation is needed before focusing? Or make the focusing optional?
(I don't use key navigation so I would just turn both off.)

**Comment by iangilman** [comment]
Indeed... the mechanism for disabling this is to use `tabIndex: ""` in your viewer options. See https://github.com/openseadragon/openseadragon/pull/805 for more info. Also see #874. It would be great if we could find a way to keep accessibility yet not have this behavior!

**Comment by leesei** [comment]
Emm, why is this issue closed?
I can still reproduce it on http://openseadragon.github.io/.

**Comment by iangilman** [comment]
Good point! Just cleaning up old questions, but it looks like this should be marked as an enhancement instead.

Any thoughts on how to solve it (i.e. how can we keep keyboard accessibility without introducing this behavior)?
